File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { SvobodaWebSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await SvobodaWebSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# SvobodaWeb SDK exists test

require "minitest/autorun"
require_relative "../SvobodaWeb_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = SvobodaWebSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
